src/main/java/com/btkelly/gnag/models/android_lint/Issues.java

```diff
@@ -0,0 +1,158 @@
+ * 
+ * <pre>
+ * &lt;complexType name="issues">
+ *   &lt;complexContent>
+ *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
+ *       &lt;sequence>
+ *         &lt;element name="issue" type="{}issue" maxOccurs="unbounded" minOccurs="0"/>
+ *       &lt;/sequence>
+ *       &lt;attribute name="format" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *       &lt;attribute name="by" type="{http://www.w3.org/2001/XMLSchema}string" />
+ *     &lt;/restriction>
+ *   &lt;/complexContent>
+ * &lt;/complexType>
+ * </pre>
+ * 
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "issues", propOrder = {
+    "issue"
+})
+@XmlRootElement(name = "issues")
+public class Issues implements Report {
+
+    protected List<Issue> issue;
+    @XmlAttribute(name = "format")
+    protected String format;
+    @XmlAttribute(name = "by")
+    protected String by;
+
+    /**
+     * Gets the value of the issue property.
+     * 
+     * <p>
+     * This accessor method returns a reference to the live list,
+     * not a snapshot. Therefore any modification you make to the
+     * returned list will be present inside the JAXB object.
+     * This is why there is not a <CODE>set</CODE> method for the issue property.
+     * 
+     * <p>
+     * For example, to add a new item, do as follows:
+     * <pre>
+     *    getIssue().add(newItem);
+     * </pre>
+     * 
+     * 
+     * <p>
+     * Objects of the following type(s) are allowed in the list
+     * {@link Issue }
+     * 
+     * 
+     */
+    public List<Issue> getIssue() {
+        if (issue == null) {
+            issue = new ArrayList<Issue>();
+        }
+        return this.issue;
+    }
+
+    /**
+     * Gets the value of the format property.
+     * 
+     * @return
+     *     possible object is
+     *     {@link String }
+     *     
+     */
+    public String getFormat() {
+        return format;
+    }
+
+    /**
+     * Sets the value of the format property.
+     * 
+     * @param value
+     *     allowed object is
+     *     {@link String }
+     *     
+     */
+    public void setFormat(String value) {
+        this.format = value;
+    }
+
+    /**
+     * Gets the value of the by property.
+     * 
+     * @return
+     *     possible object is
+     *     {@link String }
+     *     
+     */
+    public String getBy() {
+        return by;
+    }
+
+    /**
+     * Sets the value of the by property.
+     * 
+     * @param value
+     *     allowed object is
+     *     {@link String }
+     *     
+     */
+    public void setBy(String value) {
+        this.by = value;
+    }
+
+    // NON-GENERATED
+
+    @Override public boolean shouldFailBuild() {
+        return !getErrors().isEmpty();
+    }
+
+    public List<Issue> getErrors() {
+        final List<Issue> result = new ArrayList<>();
+
+        for (Issue issue : getIssue()) {
+            if (issue.getSeverity().equalsIgnoreCase("error")) {
```

Comment:
Yeah probably, the README needs some updating. I'm going to create an issue just for that.